addons/l10n_cl/views/report_invoice.xml

```diff
@@ -268,4 +268,42 @@
         </tr>
     </template>
 
+    <template id="report_invoice_document_copy" inherit_id="l10n_cl.report_invoice_document">
+        <xpath expr="//div[@id='right-elements']" position="inside">
+            <div t-if="show_information_box" name="custom_info_box" class="col-12 mt-3 p-3 border border-primary rounded">
+                <div class="row mb-2">
+                    <div class="col-12 d-flex align-items-baseline">
+                        <span class="small me-2 text-nowrap">Nombre: </span>
+                        <div class="border-bottom pb-1 flex-grow-1" stylr="min-height: 1.2rem;"/>
+                    </div>
+                </div>
+                <div class="row mb-2">
+                    <div class="col-6 d-flex align-items-baseline">
+                        <span class="small me-2 text-nowrap">RUT: </span>
+                        <div class="border-bottom pb-1 flex-grow-1" stylr="min-height: 1.2rem;"/>
+                    </div>
+                    <div class="col-6 d-flex align-items-baseline">
+                        <span class="small me-2 text-nowrap">FETCHA: </span>
+                        <div class="border-bottom pb-1 flex-grow-1" stylr="min-height: 1.2rem;"/>
+                    </div>
+                </div>
+                <div class="row mb-2">
+                    <div class="col-6 d-flex align-items-baseline">
+                        <span class="small me-2 text-nowrap">Recinto: </span>
+                        <div class="border-bottom pb-1 flex-grow-1" stylr="min-height: 1.2rem;"/>
+                    </div>
+                    <div class="col-6 d-flex align-items-baseline">
+                        <span class="small me-2 text-nowrap">FIRMA: </span>
+                        <div class="border-bottom pb-1 flex-grow-1" stylr="min-height: 1.2rem;"/>
+                    </div>
+                </div>
+                <p class="mt-3 small text-justify">
+                    El acuse de recibo que se declara en este acto, de acuerdo a lo dispuesto en 
+                    la letra b) del Art. 4° y la letra c) del Art. 5° de la Ley 19.983, acredita que 
+                    la entrega de mercaderia(s) o servicio(s)
+                </p>
+            </div>
```

Comment:
What you could do is having the first div just to have the basic space if we follow what has been done in the base template it would be. I will let you check what is an offset :smile:
```
<div t-if="..." name="..." class="col-4 offset-8">
```

now either you do multiple row and you add the border like you have you could do one big row (that was the chosen solution but i think both are fine) just multiple row will need a bit more code.

I see that you use d-flex which is nice, i would advice reading about it because you will definitely use it later in your odoo career !